src/storage/src/hummock/sstable_store.rs

```diff
@@ -57,10 +68,12 @@ impl SstableStore {
         ));
         Self {
             path,
-            store,
+            remote_store,
+            local_store,
             block_cache: BlockCache::new(block_cache_capacity),
             meta_cache,
             stats,
+            next_local_sst_id: AtomicU64::new(0),
```

Comment:
do not maintain `next_local_sst_id` in `SstableStore`.
It belongs to `Version`. Manage it in `LocalVersionManager` or other place

Reply:
Have moved it to `SharedBufferUploader`